recipes/folly/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "folly" %}
+{% set version = "2017.10.16.00" %}
+{% set sha256 = "725d98bae22c2930231773909ddd87063fe4b4dd822d65422c19bd028c2f6362" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/facebook/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 2
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
```

Comment:
Do you really need ``cmake`` and ``scons``? Because, it seems that you are using just ``autoconf``. I may be wrong, just check it please, :)

Reply:
actually not sure. I am just using that because it is on documentation.